sdk/signalr/signalr/src/hubClient.ts

```diff
@@ -0,0 +1,545 @@
+    options?: HubBroadcastOptions
+  ): Promise<RestResponse>;
+
+  public async broadcast(
+    message: string | HttpRequestBody,
+    options: HubBroadcastOptions = {}
+  ): Promise<RestResponse> {
+    const normalizedOptions = normalizeBroadcastOptions(options);
+    const { span, updatedOptions } = createSpan("SignalRClient-hub-broadcast", normalizedOptions);
+    try {
+      if (typeof message === "string") {
+        return this.client.webSocketConnectionApi.postBroadcast(
+          "text/plain",
+          message,
+          this.name,
+          updatedOptions
+        );
+      } else {
+        return this.client.webSocketConnectionApi.postBroadcast(
+          "application/octet-stream",
+          message,
+          this.name,
+          updatedOptions
+        );
+      }
+    } finally {
+      span.end();
+    }
+  }
+
+  /**
+   * Send a text message to a specific user
+   *
+   * @param username User name to send to
+   * @param message The message to send
+   * @param options Additional options
+   */
+  public sendToUser(
+    username: string,
+    message: string,
+    options?: OperationOptions
+  ): Promise<RestResponse>;
+
+  /**
+   * Send a binary message to a specific user
+   *
+   * @param username The user name to send to
+   * @param message The binary message to send
+   * @param options Additional options
+   */
+  public sendToUser(
+    username: string,
+    message: HttpRequestBody,
+    options?: OperationOptions
+  ): Promise<RestResponse>;
+  public sendToUser(
+    username: string,
+    message: string | HttpRequestBody,
+    options: OperationOptions = {}
+  ): Promise<RestResponse> {
+    const { span, updatedOptions } = createSpan("SignalRClient-hub-sendToUser", options);
+
+    try {
+      if (typeof message === "string") {
+        return this.client.webSocketConnectionApi.postSendToUser(
+          "text/plain",
+          message,
+          this.name,
+          username,
+          updatedOptions
+        );
+      } else {
+        return this.client.webSocketConnectionApi.postSendToUser(
+          "application/octet-stream",
+          message,
+          this.name,
+          username,
+          updatedOptions
+        );
+      }
+    } finally {
+      span.end();
+    }
+  }
+
+  /**
+   * Send a binary message to a specific connection
+   *
+   * @param connectionId Connection id to send to
+   * @param message The binary message
+   * @param options Additional options
+   */
+  public sendToConnection(
+    connectionId: string,
+    message: string,
+    options?: OperationOptions
+  ): Promise<RestResponse>;
+  /**
+   * Send a text message to a specific connection
+   *
+   * @param connectionId Connection id to send to
+   * @param message The text message
+   * @param options Additional options
+   */
+  public sendToConnection(
+    connectionId: string,
+    message: HttpRequestBody,
+    options?: OperationOptions
+  ): Promise<RestResponse>;
```

Comment:
Do you have 'binary' and 'text' around the wrong way in your docs for these two methods?